### PR TITLE
EES-2270 EES-2271 add owner and adopter publications to methodology summary

### PR DESCRIPTION
**Pull request by amyb-hiveit**
This is hidden under a flag so won't affect the current version,  switch showPublications to true to see it in action.
Screenshot (24)

### CODE REVIEW

File: src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx [hunk 1]
```diff
@@ -44,6 +86,45 @@ const MethodologySummaryPage = ({
                   'Not yet published'
                 )}
               </SummaryListItem>
+              {currentMethodology.publication && (
+                <SummaryListItem term="Publication">
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx:90]: Might be better to call this 'Owning publication'?

File: src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx [hunk 2]
```diff
@@ -44,6 +86,45 @@ const MethodologySummaryPage = ({
...
+                      },
+                    )}
+                  >
+                    {currentMethodology.publication.title}
+                  </Link>
+                </SummaryListItem>
+              )}
+              {currentMethodology.otherPublications &&
+                currentMethodology.otherPublications.length > 0 && (
+                  <SummaryListItem term="Other publications using this methodology">
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx:106]: This seems like a bit of a mouthful as we're already in the context of a methodology. Would suggest calling this 'Other publications' (combined with the other suggestion for 'Owning publication') instead?

File: src/explore-education-statistics-admin/src/services/methodologyService.ts [hunk 3]
```diff
@@ -6,6 +6,12 @@ export type UpdateMethodology = {
 
 export type MethodologyStatus = 'Draft' | 'Approved';
 
+type MethodologyPublication = {
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/services/methodologyService.ts:9]: For consistency with the majority of types, would use an interface instead

File: src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx [hunk 4]
```diff
@@ -11,18 +11,60 @@ import SummaryListItem from '@common/components/SummaryListItem';
...
   match,
 }: RouteComponentProps<MethodologyRouteParams>) => {
   const { methodologyId } = match.params;
 
-  const { value: currentMethodology, isLoading } = useAsyncHandledRetry(() =>
+  const { value: realMethodology, isLoading } = useAsyncHandledRetry(() =>
     methodologyService.getMethodology(methodologyId),
   );
 
+  const myFakeMethodology = {
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx:31]: Could hoist this variable out of the component scope to save having to build this object on every render

**Review by ntsim — CHANGES_REQUESTED** [review]

**Review by ntsim — APPROVED** [review]

**Review by benoutram — APPROVED** [review]

**Pull request merged**